ruby/db/mysql/ib_buffer_pool

```diff
@@ -0,0 +1,253 @@
+4294967278,284
```

Comment:
Do we need all this files to be committed?

Reply:
Thx, looks like we should add them to the gitignore.